src/main/java/pl/wedding/management/service/weddinghall/model/WeddingHall.java

```diff
@@ -0,0 +1,15 @@
+package pl.wedding.management.service.weddinghall.model;
+
+import lombok.*;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class WeddingHall {
+    private Set<TypesOfHalls> typesOfHalls = EnumSet.allOf(TypesOfHalls.class);
```

Comment:
Shouldn't this be populated with only certain type of hall, not all of them at once?

Reply:
This should be empty set.